storage/statedb/trie.go

```diff
@@ -37,6 +37,11 @@ var (
 	emptyState = crypto.Keccak256Hash(nil)
 )
 
+// TrieOpts consists of trie operation options
+type TrieOpts struct {
+	Prefetching bool // If true, certain metric is enabled
```

Comment:
What do you mean `certain metric` here?

Reply:
The only place `prefetching` flag is relevant is https://github.com/klaytn/klaytn/blob/v1.10.2/storage/statedb/trie.go#L508 where memcacheCleanPrefetchMissMeter ("trie/memcache/clean/prefetch/miss") is updated. "certain metric" refers to this.

Reply:
Okay. Thanks.